Documentation/tutorial/intro_toc.md

```diff
@@ -0,0 +1,99 @@
+Table-Of-Content(TOC) Files
+===========================
+
+Introduction
+---------------------
+DocFX is able to process YAML files, which we call *Metadata File*s, and Markdown files, which we call *Conceptual File*s. Besides that, DocFX introduces a way to organize these files using *Table-Of-Content File*s, which we also call *TOC File*s, so that users can navigate through *Metadata File*s and *Conceptual File*s.
+
+*TOC File*s must have file name `toc.md` or `toc.yml`, notice that file name is case-insensitive.
+
+Basic format
+----------------
+### Markdown format TOC `toc.md`
+
+`toc.md` leverage Markdown [Atx-style headers](http://daringfireball.net/projects/markdown/syntax#header) which use 1-6 hash characters at the start of the line to represent the TOC levels 1-6. We call each line starting with hash characters a *TOC Item*. A *TOC Item* with higher level is considered as the child of the nearest upper *TOC Item* with less level. A sample `toc.md` is as below:
+
+```md
+# [Header1](href)
+## [Header1.1](href)
+# Header2
+## [Header2.1](href)
+### [Header2.1.1](href)
+## [Header2.2](href)
+```
+
+For a *Toc Item*, it can be either plain text or a [Markdown inline link](http://daringfireball.net/projects/markdown/syntax#link). 
+
+Three kinds of links are supported:
+
+1. Absolute path, for example, `http://example.net`.
```

Comment:
For breadcrumb TOC in DOCS, `/example/` is also seen as absolute path. Should it be added here?